kernel/vars.mli

```diff
@@ -140,4 +140,6 @@ val subst_univs_level_context : Univ.universe_level_subst -> Constr.rel_context
 val subst_instance_constr : Instance.t -> constr -> constr
 val subst_instance_context : Instance.t -> Constr.rel_context -> Constr.rel_context
 
+val univ_instantiate_constr : Instance.t -> constr univ_abstracted -> constr
```

Comment:
This is a dangerous function because it drops the universe constraints, and nothing indicates this.

Reply:
>This is a dangerous function because it drops the universe constraints, and nothing indicates this.

Should we just go back to carrying an `int` then.

Reply:
A comment is enough!